packages/app/src/app/pages/Sandbox/Editor/Workspace/Dependencies/ExternalFonts/index.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+
+import CrossIcon from 'react-icons/lib/md/clear';
+
+import { EntryContainer, IconArea, Icon } from '../../elements';
+import { Link } from '../elements';
+
+const getFamily = search => {
+  const hashes = search.slice(search.indexOf('?') + 1).split('&');
+  const params = {};
+  hashes.map(hash => {
```

Comment:
You can use `reduce` instead of `map` here

```ts
const params = hashes.reduce((paramsObj, hash)=> {
  const [key, val] = hash.split('=');
  
  return {
    ...paramsObj,
    [key]: decodeURIComponent(val),
  };
}, {});
```